Makefile

```diff
@@ -28,10 +28,10 @@ core_static:
 
 .PHONY: core_generate_all
 core_generate_all:
-	go install github.com/dmarkham/enumer@v1.5.9
-	go install github.com/rakyll/statik@v0.1.7
-	go install go.uber.org/mock/mockgen@v0.3.0
-	go install golang.org/x/tools/cmd/stringer@v0.15.0
+	go install github.com/dmarkham/enumer@v1.5.9 		# https://pkg.go.dev/github.com/dmarkham/enumer?tab=versions
+	go install github.com/rakyll/statik@v0.1.7			# https://pkg.go.dev/github.com/rakyll/statik?tab=versions
+	go install go.uber.org/mock/mockgen@v0.4.0			# https://pkg.go.dev/go.uber.org/mock/mockgen?tab=versions
+	go install golang.org/x/tools/cmd/stringer@v0.1t.0	# https://pkg.go.dev/golang.org/x/tools/cmd/stringer?tab=versions
```

Comment:
```suggestion
	go install golang.org/x/tools/cmd/stringer@v0.17.0	# https://pkg.go.dev/golang.org/x/tools/cmd/stringer?tab=versions
```

Reply:
looking at the provided link i believe this should have been 0.17.0

Reply:
oops